.github/workflows/pull_request.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        fdbver: [ "6.2.30", "6.3.22", "7.1.0-rc3" ]
+        fdbver: [ "6.2.30", "6.3.22", "7.1.0" ]
```

Comment:
We should review if we really want to run everything in those settings or if we can reduce the number of runs.